### ISSUE TITLE
Allow listing of all plugins and themes via WP CLI

### ISSUE THREAD

**Issue by cohan** [mainwp/mainwp#62]
Could we have functionality to list all themes and plugins (maybe use --list-all?) when running wp mainwp plugin --list rather than just those that need updates?

**Comment by bogdan-mainwp** [comment]
The dev team will add this for the next release.

**Comment by bogdan-mainwp** [comment]
Included in 3.4.3 version. 

Available commands:
```
wp mainwp plugin --list-all
wp mainwp plugin 2,5 --list-all

wp mainwp theme --list-all
wp mainwp theme 2,5 --list-all
```

_(2, 5 == child site IDs)_
